### ISSUE TITLE
wqp sites associated with multiple site_ids

### ISSUE THREAD

**Issue by limnoliver** [USGS-R/lake-temperature-model-prep#50]
There seems to be two issues:

1) we have multiple polygons for the same lake which was caused by border issues when the MGLP footprint expansion was included (e.g., polygon for the same lake in both the winslow NHD shapefile and the MGLP shapefil).

2) Issues within the NHD such that there are overlapping shapefiles for water bodies. If the lat/long for a WQP site falls in that overlap, it gets two site IDs: 
![image](https://user-images.githubusercontent.com/15788176/57031775-3bcad000-6c0e-11e9-8cfc-872bc2fada86.png)

This creates duplication issues downstream when wants-haves-needs are being created. In these cases, it seems that the site will be pulled multiple times from WQP, but in the end, WQP munging gets rid of duplicated records via `distinct()`. So, that part isn't a huge deal, but assigning the value to the "right" polygon is still an issue.

**Comment by jzwart** [comment]
So it looks like there will be 4 WQP sites with multiple `nhd_ids`? from the panel graph you provide, it looks like there are 4 unique locations
1) `nhd_120052142` , `nhd_120052865`
2) `nhd_120052896`, `nhd_120053307`, `nhd_13118872`
3) `nhd_167122141`, `nhd_167122172` 
4) `nhd_167204344`, `nhd_167800983`

**Comment by limnoliver** [comment]
Not exactly. For example, the big lake you're seeing above is Lake Winnibogoshis, which has multiple sites and the shape files (nhd_120053307 and nhd_13118872) are essentially identical, so there are probably a lot of sites from that lake that get two `site_id`s.

**Comment by jread-usgs** [comment]
curious if we'll have this issue w/ NHD HR #53 . Is there code used to figure this out for the previous NHD that we could re-run to see if this problem is still around?
